src/server/scripts/Outland/TempestKeep/Mechanar/boss_pathaleon_the_calculator.cpp

```diff
@@ -48,8 +50,13 @@ enum Spells
 enum Misc
 {
     ACTION_BRIDGE_MOB_DEATH = 1, // Used by SAI
+    ITEM_WEAPON             = 23382,
+    ITEM_SHIELD             = 24328,
+    ITEM_FRENZY             = 29455,
 };
 
+bool _isEnraged;
```

Comment:
this needs to be inside the boss class, otherwise it becomes a global variable

Reply:
moved it down in the struct, hope that fixes it :P